packages/gestalt/src/ButtonSocial.tsx

```diff
@@ -0,0 +1,325 @@
+import { forwardRef, useImperativeHandle, useRef } from 'react';
+import getAriaLabel from './accessibility/getAriaLabel';
+import NewTabAccessibilityLabel from './accessibility/NewTabAccessibilityLabel';
+import Box from './Box';
+import VRButtonSocial from './ButtonSocial/VRButtonSocial';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider';
+import Flex from './Flex';
+import Icon from './Icon';
+import InternalLink from './Link/InternalLink';
+import Text from './Text';
+import useInExperiment from './useInExperiment';
+
+const TEXT_OPTIONS = {
+  1: 'Login with',
+  2: 'Continue with',
+  3: 'Sign up with',
+};
+
+const SERVICES_OPTIONS = {
+  1: 'Apple',
+  2: 'Facebook',
+  3: 'Google',
+  4: 'Email',
+};
+
+type ButtonProps = {
+  /**
+   * Label to provide more context around ButtonLink’s function or purpose. See the [Accessibility guidelines](/foundations/accessibility) to learn more.,
+   */
+  accessibilityLabel?: string;
+  /**
+   * Available for testing purposes, if needed. Consider [better queries](https://testing-library.com/docs/queries/about/#priority) before using this prop.
+   */
+  dataTestId?: string;
+  /**
+   * Indicates whether this component is hosted in a light or dark container.
+   * Used for improving focus ring color contrast.
+   */
+  focusColor?: 'lightBackground' | 'darkBackground';
+  /**
+   * Default Buttons are sized by the text within the ButtonSocial whereas full-width Buttons expand to the full width of their container.
+   */
+  fullWidth?: boolean;
+  /**
+   * Use "-1" to remove ButtonSocial from keyboard navigation. See the [Accessibility guidelines](/foundations/accessibility) to learn more.
+   */
+  tabIndex?: -1 | 0;
```

Comment:
let's remove this tabIndex... I dont think for these buttons this might be  ever needed